fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalSetOperation.java

```diff
@@ -130,4 +137,41 @@ public int getArity() {
         return children.size();
     }
 
+    @Override
+    public boolean pushDownRuntimeFilter(CascadesContext context, IdGenerator<RuntimeFilterId> generator,
```

Comment:
it is not convivent to write the rf pushing down in current runtime filter generator visit framework, since the rf producer is the join node, but it can be pushed down into the very deep scan node, inside different layer of subquery, set op, etc.  Seems transferMap is not enough, which only maintains the probeExpr's visibility, but not provide the facility for the pushing down between different plan tree layer.